react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { readItems, type Item } from './read';
+
+export function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    readItems();
```

Comment:
Is this leftover from earlier development? It shouldn't be here. It doesn't affect the behavior of the code, but it does make an extra, unnecessary, call to readItems.